ios/engine/KMEI/KeymanEngine/Classes/kmp.json/KMPMetadata.swift

```diff
@@ -42,6 +42,32 @@ class KMPMetadata: Codable {
     case Unsupported
   }
 
+  init(from resource: LanguageResource) {
```

Comment:
Can you add a comment as to how this transform works and its purpose?

Things that stick out:
* the fact that there will only ever be a single keyboard:language pair (or model:language pair) in the LanguageResource.
* keyboard version / model version will always be 0.0.0?
* is it intended for use with migration from old cloud-installed .js keyboards?